.github/workflows/get-new-package-version/action.yml

```
@@ -12,8 +12,11 @@ runs:
     - shell: bash
       run: |
             CLEAN_PACKAGE_VERSION=$(node -p "require('./package.json').version")
-            PACKAGE_VERSION=${CLEAN_PACKAGE_VERSION}-${{ inputs.CHANNEL }}-rc
-
+            # If the channel is not `daily-cloud`, than add `-rc` to the the package version
+            if [[ ${{ inputs.CHANNEL }} != "daily-cloud" ]]; then
```

This condition will be changed once we move this to action to production.
It will be:
```suggestion
            if [[ ${{ inputs.CHANNEL }} != "cloud" ]]; then
```